### PR TITLE
fix: inconsistent error message.

### PR DESCRIPTION
**Pull request by felsweg-iota**
# Description of change

This fixes an inconsistent error message when trying to access a store location.

## Links to any relevant issues

none. 

## Type of change

Choose a type of change, and delete any options that are not relevant.

- [x] Bug fix (a non-breaking change which fixes an issue)

## How the change has been tested

unit and integration tests

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by nothingismagick — CHANGES_REQUESTED** [review]
This makes good sense. Please add a .changes file for this :)

**Review by nothingismagick — APPROVED** [review]

File: client/src/actors/secure.rs [hunk 1]
```diff
@@ -65,6 +65,12 @@ pub enum VaultError {
     AccessError,
 }
 
+#[derive(DeriveError, Debug)]
+pub enum StoreError {
+    #[error("Unable to read from store")]
```
Comment (elenaf9) [inline client/src/actors/secure.rs:70]:
```suggestion
    #[error("The store does not exist")]
```
Wouldn't we want to provide the info **why** it is _"Unable to read from store"_, ig. that is does not exist?

Comment (felsweg-iota) [reply client/src/actors/secure.rs:70]: This fix re-integrates the error message from before, and restores some "expected behavior"*. I would suggest to work on more proper error types. Eventually increasing the cognitive value of this one. So, I do agree. 

* on a functional level the behavior itself has not changed, but the former error message created some confusion, hence this quick fix.

**Review by elenaf9 — APPROVED** [review]
LGTM

**Pull request merged**